cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -383,6 +383,16 @@ def accelerator_experiments(self, key, value):
     return _extensions
 
 
+@model.over("_migration", "^536__")
+@out_strip
+def open_access(self, key, value):
+    """Translate open access field."""
+    if "r" in value:
```

Comment:
honestly it would be great to add some explanations, because this was not easy. What's inside the `r` and why if defined means `open_access`?
Comment or better naming.